lucene/core/src/java/org/apache/lucene/codecs/compressing/LZ4WithPresetDictCompressionMode.java

```diff
@@ -14,12 +14,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.lucene.codecs.lucene87;
+package org.apache.lucene.codecs.compressing;
```

Comment:
It seems like these classes should also be duplicated into 87 and 90 versions, with the 90 versions living in the `lucene90.compressing` package. In other places we decided not to share encoder/ decoder helper classes, for example we duplicated `ForUtil` instead of sharing it across postings formats.

Reply:
Ok, I copy classes to backwards codec so we are not sharing any of those classes